pkg/requestdecorator/requestdecorator.go

```diff
@@ -0,0 +1,170 @@
+package requestdecorator
```

Comment:
Can you copy `README.md` here as `Package requestdecorator...`. To parse it with godoc